### PR TITLE
Feature/intel 910

### PR DESCRIPTION
**Pull request by hawk-luca-ricchi**

### CODE REVIEW

File: .github/workflows/build-python-ai-scores.yaml [hunk 1]
```diff
@@ -50,7 +53,7 @@ jobs:
       - uses: actions/checkout@v3
       - uses: actions/setup-python@v4
         with:
-          python-version: 3.8
+          python-version: ${{ python-version }}
```
Comment (hawk-luca-ricchi) [inline .github/workflows/build-python-ai-scores.yaml:56]: @hawk-sergey-kurilo this will cause ai-scores to break if we don't specify the python version on the caller workflows inside the ai-scores repository. It's up to you to revert this change or contact @hawk-sarthak-garg to add it in ai-scores

File: .github/workflows/build-python.yaml [hunk 2]
```diff
@@ -83,7 +83,7 @@ jobs:
           pipenv install --dev
           pipenv lock --keep-outdated > pipenv-requirements.txt
           pipenv requirements > requirements.txt
-          pipenv run pytest tests --junitxml=report.xml
+          pipenv run tests --junitxml=report.xml
```
Comment (hawk-luca-ricchi) [inline .github/workflows/build-python.yaml:86]: @hawk-sergey-kurilo about this I have no knowledge at all. Other repositories using this workflow are ai-data-sink and entity-resolution. If you are sure that this change is ok for all of them we can merge it

Comment (hawk-sergey-kurilo) [reply .github/workflows/build-python.yaml:86]: this is ok

File: .github/workflows/build-python-ai-scores.yaml (hunk 1 above)
Comment (hawk-sergey-kurilo) [reply .github/workflows/build-python-ai-scores.yaml]: reverted this change

**Review by hawk-sergey-kurilo — APPROVED** [review]
